misk/src/main/java/misk/client/README.md

```diff
@@ -0,0 +1,2 @@
+We include two java classes here as they're direct copies from okhttp's
```

Comment:
I had issues porting these over as kotlin, I think primarily due to an issue with the jnr-unixsocket library. This change I think should help, but in the meantime I think these direct copies are fine (https://github.com/jnr/jnr-unixsocket/pull/62)